.gitignore

```diff
@@ -1,2 +1,127 @@
-# Python compiled files
-*.pyc
+### Python ###
```

Comment:
did you use https://www.gitignore.io/? might as well add (linux, vi, emacs, windows) sections from there, while you're at it.

Reply:
Yeah, straight from the Python one. Will update it with those extra ones 😄